src/App.jsx

```diff
@@ -1,10 +1,47 @@
 import './App.css';
+import axios from 'axios';
+import worldMap from "./assets/world_map.png";
+import {useState} from "react";
+import {sort} from "./assets/helpers/sort.jsx";
+import Search from "./pages/search.jsx";
 
 function App() {
 
+
+    const [countries, setCountries] = useState([])
+    sort(countries)
+
+
+    async function fetchInfo() {
+        try {
+            const result = await axios.get('https://restcountries.com/v3.1/all');
+            setCountries(result.data);
+        } catch (e) {
+            console.error(e);
+        }
+    }
+
+
     return (
         <>
-            Maak je applicatie hier!
+            <Search></Search>
+            <div className="header">
+                <img src={worldMap} alt="map of the world"/>
+                <h2>World Regions</h2>
+                <button type="button" onClick={fetchInfo}>Get info</button>
+            </div>
+            <ul className="countryList">
+                {countries.map((country) => {
```

Comment:
Je had hier spreading kunnen toepassen, zodat je niet overal `country. ` vóór hoeft te zetten
```suggestion
                {countries.map(({ name, flags, region, population }) => {
```